roles/openshift_health_checker/openshift_checks/logging/logging.py

```diff
@@ -20,7 +21,7 @@ def is_active(cls, task_vars):
     @staticmethod
     def is_first_master(task_vars):
```

Comment:
Thanks, updated method to only check if first master. Moved `logging_deployed` check to `is_active`